setup.py

```diff
@@ -32,7 +32,8 @@
     install_requires=[
         "bravado-core >= 4.2.2",
         "crochet >= 1.4.0",
-        "fido >= 2.1.0, < 3.2.0",
+        "fido >= 2.1.0, <= 3.2.0",
```

Comment:
This is kind of a weird restriction. Do we know that bravado is incompatible with fido 3.2.1? Otherwise I would specify this with "< 3.3" and expect fido not to break compatibility in minor / patch releases. We could also simply remove this restriction and only add it back in if a known incompatibility comes up. We're not restricting other libraries in this way either.